src/sentry/static/sentry/app/components/discoverButton.tsx

```diff
@@ -13,8 +16,29 @@ type Props = React.PropsWithChildren<{
  * doesn't have access to discover results.
  */
 function DiscoverButton({children, ...buttonProps}: Props) {
+  const noFeatureMessage = t('Requires discover feature.');
```

Comment:
I think I want to clean up FeatureDisabled so it doesn't require specifying the feature message at each usage point, but this is probably fine for now.

Reply:
This copy would show up on on-premise (see first screenshot).

Reply:
right. I'm just saying that I wish we didn't need to write out the plain-english name of the feature in each usage of FeatureDisabled.